can/io/logger.py

```diff
@@ -325,6 +325,12 @@ def __init__(
         self.base_filename = os.path.abspath(base_filename)
         self.max_bytes = max_bytes
 
+        # BLFWriter specific
+        _128_kb = 128 * 1024
+        if kwargs.get('max_container_size', 1) and max_bytes > 0:
+            kwargs['max_container_size'] = \
+                _128_kb if max_bytes > _128_kb else max_bytes * 0.1
+
```

Comment:
@zariiii9003 This would get rid of the nuance. I'm anticipating that you are not going to like the `BLFWriter` specific code inside of the `SizedRotatingLogger` constructor, but let me know.

Reply:
Yes, i don't like this here 😄 
Also i do not understand what this does tbh, what's the purpose of this? And why is the default value of `kwargs.get('max_container_size', 1)`  `=1`?

Reply:
If `max_container_size` is not defined then set the `max_container_size`. If `max_container_size` is not defined then set the value to be `128 kb` unless the `file_size` is less than `128 kb`. In that case set the `max_container_size` to 10% of the `file_size`. 

I'm reverting the commits because I agree that its not a good location.